rules_building_block/credential_access_mdmp_file_creation.toml

```diff
@@ -35,13 +35,16 @@ file where host.os.type == "windows" and event.type == "creation" and
 
   (
     (
+      process.name : "System" or
```

Comment:
should we be tuning this one either, based on description?

Reply:
I think any rule can use some tuning to exclude common system activities